package.json

```diff
@@ -30,32 +30,26 @@
       "project": "./tsconfig.json"
     }
   },
+  "eslintIgnore": ["**/demoapp-react/**"],
   "scripts": {
     "del-dist": "lerna exec --parallel -- del-cli dist",
     "prebuild": "yarn del-dist",
-
     "build:react-core:build": "lerna exec --scope @map-colonies/react-core -- yarn build",
     "build:react-core:postbuild": "lerna exec --scope @map-colonies/react-core -- yarn postbuild",
     "build:react-core": "yarn build:react-core:build && yarn build:react-core:postbuild",
     "build:react": "lerna exec --scope @map-colonies/react-components -- yarn build",
-    "build:all": "lerna run build --npm-client=yarn",
-
+    "build:all": "yarn build:react-core && yarn build:react",
```

Comment:
lerna as a cross packages tool might be considered rather than lerna scoped script (like in publish) 
Up to you.